articles/most-useful-node.js-packages/index.md

```diff
@@ -0,0 +1,349 @@
+Here a few examples of date objects using MomentJS: To use MomentJs with Node.js, install the module using command `npm install moment`. Then simply use `require ()` in your application as in the following example:
+```js
+const moment = require('moment');
+const today = moment();
+console.log(today.format());
+```
+**OUTPUT: 2020-08-27T09:21:49+01:00**
+
+MomentJS can still run from the browser. It creates a global moment object, which is used to access date and time parsing and manipulation functionality. Include `<script>` as in the example below.
+```js
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset ="UTF-8"></meta>
+        <title> MomentJS Date Object </title>
+    </head>
+    <body>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js"></script>
+        <script>
+        // You have a 'moment' global here
+         const today = moment();
+         console.log(today.format());
+        </script>
+    </body>
+</html>
+```
+MomentJS is an awesome Node.js package. It simplifies date and time related parsing, manipulation as well as validation. If you have a Node.js application that uses data and time objects, consider trying MomentJS. It will help you to solve any date problem in your application. MomentJS support international languages in you date objects which is a good thing to have in your application if you are dealing with multiple language application.
+
+**Other validations and formatting libraries include:**
+-   [`Validator.js`](https://github.com/validatorjs/validator.js) - A library for string validators  and sanitizer.
+-   [`Express-validator`](https://www.npmjs.com/package/express-validator) -
+    It is an Express Middleware for Validator.js.
+### 6. Request
+Request is a simplified HTTP client that makes it possible to make `http` calls. It supports HTTPS and follows redirect by default. Request is a fantastic choice if you want an easy to use library to deal with HTTP request in a sane way. If you want to use promises, you can check out the request-promise library. It is used to describe a request to a server and get a response.
+
+**Request has the following main contractors:**
+-   **Body** - The data to send with the request.
+-   **Cache** - A sting with the following instanceS: default, force-cache, no-cache, no-store, only-if-cached, reload etc.
+-   **Credentials** - A sting which include omit, same-origin instance properties.
+-   **Headers** - It is the `http` headers to send with the request. It is passed to the headers contractor.
+-   **Method** - HTTP method such as `GET` `POST` `DELETE`, `PUT`.
+-   **Mode** - A string with one of same-origin, no-cors, cors instances.
+
+**Examples of simple Request**
```

Comment:
of **A**

Reply:
Could be simplified now to Restify Examples, @kimkimani.

Reply:
@louisefindlay23 I didn't understand this comment.